types/echarts/options/legend.d.ts

```diff
@@ -422,6 +453,194 @@ declare namespace echarts {
+                 * @default false
+                 */
+                show?: boolean;
+                /**
+                 * Distance to the host graphic element. Works when position is string value (like 'top'、'insideRight').
+                 *
+                 * @default 5
+                 *
+                 * @see https://echarts.apache.org/examples/zh/editor.html?c=doc-example/label-position
+                 */
+                distance?: number;
+                /**
+                 * Rotate label, from -90 degree to 90, positive value represents rotate anti-clockwise.
+                 *
+                 * @see https://echarts.apache.org/examples/zh/editor.html?c=bar-label-rotation
+                 */
+                rotate?: number;
+                /**
+                 * text color.
+                 * If set as 'auto', the color will assigned as visual color, such as series color.
+                 *
+                 * @default '#fff'
+                 */
+                color?: Color;
+                /**
+                 * font style.
+                 * Options are:'normal','italic','oblique'
+                 *
+                 * @default 'normal'
+                 */
+                fontStyle?: 'normal' | 'italic' | 'oblique';
+                /**
+                 * font thick weight.
+                 *
+                 * @default 'normal'
+                 */
+                fontWeight?:
+                    | 'normal'
+                    | 'bold'
+                    | 'bolder'
+                    | 'lighter'
+                    | 100
+                    | 200
+                    | 300
+                    | 400
+                    | 500
+                    | 600
+                    | 700
+                    | 800
+                    | 900;
+                /**
+                 * font family.
+                 *
+                 * @default 'sans-serif'
+                 */
+                fontFamily?: string;
+                /**
+                 * font size
+                 *
+                 * @default 12
+                 */
+                fontSize?: number;
+                /**
+                 * Horizontal alignment of text, automatic by default.
+                 */
+                align?: 'left' | 'center' | 'right';
+                /**
+                 * Vertical alignment of text, automatic by default.
+                 */
+                verticalAlign?: 'top' | 'middle' | 'bottom';
+                /**
+                 * Line height of the text fragment.
+                 */
+                lineHeight?: number;
+                /**
+                 * Background color of the text fragment.
+                 * Can be color string, like '#123234', 'red', 'rgba(0,23,11,0.3)'.
+                 * Or image can be used, for example:
+                 * @example
+                 * ```ts
+                 * backgroundColor: {
+                 * 	image: 'xxx/xxx.png'
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/echarts/options/legend.d.ts;
  Line: 541;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.